src/main/java/com/twilio/oai/resolver/ruby/RubyCaseResolver.java

```diff
@@ -1,17 +1,17 @@
 package com.twilio.oai.resolver.ruby;
 
-import com.twilio.oai.StringHelper;
 import com.twilio.oai.resolver.CaseResolver;
+import org.openapitools.codegen.utils.StringUtils;
 
 public class RubyCaseResolver implements CaseResolver {
     @Override
     public String productOperation(final String product) {
-        return StringHelper.toSnakeCase(product);
+        return StringUtils.underscore(product);
```

Comment:
Is there a reason for this change? The StringHelper implementation was written to handle some nuisances in our APIs that StringUtils wasn't handling the same way.

Reply:
Not really, just wanted the underscore to be appended. Tested & found that toSnakeCase would also work in this case.